airbyte-integrations/connectors/source-posthog/source_posthog/manifest.yaml

```diff
@@ -152,13 +152,12 @@ definitions:
           inject_into: "request_parameter"
           field_name: "limit"
         pagination_strategy:
-          type: "CursorPagination"
-          cursor_value: "{{ response['next'] }}"
+          type: "OffsetIncrement"
           page_size: 10000
         page_token_option:
-          type: RequestPath
-        $parameters:
-          url_base: "#/definitions/requester/url_base"
+          type: RequestOption
```

Comment:
I don't particularly mind, but generally cursor should be better than offset/increment. Can you link me to posthog docs explaining whether that stream does / does not have the cursor?

Reply:
Hypothetically, the response.next _should_ be there.

Reply:
Ah I see.